README.rst

```diff
@@ -1,34 +1,38 @@
-PathCORE
---------
+PathCORE-T
+----------
 Python 3 implementation of methods described in
-`Chen et al.'s 2017 PathCORE paper <https://doi.org/10.1101/147645>`_ for
+`Chen et al.'s 2017 PathCORE-T paper <https://doi.org/10.1101/147645>`_ for
 identifying pathway-pathway interactions using features constructed from
 transcriptomic data.
 
 This code has been tested on Python 3.5.
 The documentation for the modules in the package can be
 `accessed here <http://pathcore-demo.herokuapp.com/static/data/docs_pathcore/index.html>`_.
 
+Note that this software was renamed from PathCORE to PathCORE-T in Oct 2017.
```

Comment:
I think it will be good to add what the `T` stands for. I am wondering - why the name change???

Reply:
Haha, good point. :) I'll add that. (PathCORE identifies **Path**way CO-occurrence RElationships from features extracted from **T**ranscriptomic data.) Thanks Greg!